r/R/dataset-write.R

```diff
@@ -53,6 +53,34 @@
 #' - `codec`: A [Codec] which will be used to compress body buffers of written
 #'   files. Default (NULL) will not compress body buffers.
 #' @return The input `dataset`, invisibly
+#' @examples
+#' dout1 <- paste0(tempdir(), "/direct")
+#' dout2 <- paste0(tempdir(), "/group_by")
+#'
+#' # partitioning ----
+#' write_dataset(mtcars, dout1, "feather",
+#'  partitioning = "cyl",
```

Comment:
It might be nice to have a two-variable partitioning scheme here instead of just one, that way it shows off that this can be any number of column names / groups in group_by()